dlrover/python/unified/api/builder/base.py

```diff
@@ -96,41 +99,40 @@ def get_workload(self, role):
 
     def submit(
         self,
-        job_name,
+        job_name: str | None = None,
```

Comment:
keep or remove this field? the annotation of 'job_name' below has been removed:
![Uploading image.png…]()